invenio_theme_tugraz/templates/invenio_theme_tugraz/accounts/login.html

```diff
@@ -1,74 +1,76 @@
 {#
   Copyright (C) 2020 TUGRAZ
-  Copyright (C) 2020 mojib wali.
+  Copyright (C) 2020 mojib wali
+  Copyright (C) 2020 Nikita Lvov
 
   invenio-theme-tugraz is free software; you can redistribute it and/or
   modify it under the terms of the MIT License; see LICENSE file for more
   details.
 #}
-
-{%- extends "invenio_theme/page_cover.html" %}
+{%- extends config.ACCOUNTS_COVER_TEMPLATE %}	
 {% from "invenio_accounts/_macros.html" import render_field, form_errors %}
 
+
 {% block page_body %}
-  <div class="ui padded segments big form">
-    <div class="ui segment padded">
-      <div class="divider hidden"></div>
+<div class="ui centered stackable grid">
```

Comment:
```suggestion
<div class="ui container">
<div class="ui centered stackable grid">
...
```
Add the div inside a container, we need some padding at bottom see screenshot.
![have a look](https://user-images.githubusercontent.com/44528277/88392936-261a6b00-cdbd-11ea-868e-82a1d8503e04.PNG)